tests/sources/math_transpose_perf.cpp

```diff
@@ -0,0 +1,127 @@
+// SPDX-FileCopyrightText: © 2025 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cstdint>
+
+#include "build.h"
+#include "ckernel.h"
+#include "ckernel_defs.h"
+#include "llk_defs.h"
+#include "params.h"
+#include "perf.h"
+#include "profiler.h"
+#include "tensix_types.h"
+
+// Globals
+uint32_t unp_cfg_context          = 0;
+uint32_t pack_sync_tile_dst_ptr   = 0;
+uint32_t math_sync_tile_dst_index = 0;
+
+#ifdef LLK_TRISC_UNPACK
+
+#include "llk_unpack_A.h"
+#include "llk_unpack_common.h"
+
+void run_kernel()
+{
+    volatile uint32_t* const src_a = reinterpret_cast<volatile uint32_t*>(0x1a000);
+    volatile uint32_t* const src_b = reinterpret_cast<volatile uint32_t*>(0x1e000);
+
+    {
+        ZONE_SCOPED("INIT")
+        _llk_unpack_A_hw_configure_<is_fp32_dest_acc_en, StochRndType::None, unpack_to_dest>(
+            formats.unpack_src, formats.unpack_dst, FACE_R_DIM, false, TILE_NUM_FACES);
+        _llk_unpack_A_init_<BroadcastType::NONE, false, EltwiseBinaryReuseDestType::NONE, unpack_to_dest>(
+            UNPACK_TRANSPOSE_FACES, false, FACE_R_DIM, TILE_NUM_FACES, formats.unpack_src, formats.unpack_dst);
+        ckernel::tensix_sync(); // -> perf
+    }
+
+    {
+        ZONE_SCOPED("TILE_LOOP")
+        for (uint32_t tile = 0; tile < TILE_CNT; tile++)
+        {
+            _llk_unpack_A_<BroadcastType::NONE, false, EltwiseBinaryReuseDestType::NONE, unpack_to_dest>(
+                L1_ADDRESS(src_a + (tile % 8) * 0x1000), UNPACK_TRANSPOSE_FACES, formats.unpack_src, formats.unpack_dst);
+            _llk_unpack_set_srcb_dummy_valid_();
+        }
+        ckernel::tensix_sync(); // -> perf
+    }
+}
+
+#endif
+
+#ifdef LLK_TRISC_MATH
+
+#include "llk_math_common.h"
+#include "llk_math_eltwise_unary_datacopy.h"
+#include "llk_math_transpose_dest.h"
+
+void run_kernel()
+{
+    constexpr bool is32 = is_32bit((DataFormat)formats.math);
```

Comment:
```suggestion
    constexpr bool is_32bit_format = is_32bit(static_cast<DataFormat>(formats.math));
```
Using is_32bit_format would make the code clearer for anyone reading or maintaining it.